agent/Cargo.toml

```diff
@@ -18,13 +18,13 @@ tracing-opentelemetry = "0.18"
 
 # k8s-openapi must match the version required by kube and enable a k8s version feature
 k8s-openapi = { version = "0.17.0", default-features = false, features = ["v1_20"] }
-kube = { version = "0.78.0", default-features = true, features = [ "derive", "runtime", "rustls-tls" ] }
+kube = { version = "0.78.0", default-features = false, features = [ "derive", "runtime", "rustls-tls" ] }
 
 semver = { version = "1.0", features = [ "serde" ] }
 serde = { version = "1", features = [ "derive" ] }
 serde_json = "1"
 snafu = "0.7"
 tokio = { version = "1", features = ["macros", "rt-multi-thread", "time"] }
-reqwest = { version = "0.11", features =  [ "json" ] }
```

Comment:
I think we have to enable feature `rustls-tls` here? Or are we just using `reqwest` to talk to the local Bottlerocket API socket?

Reply:
Nice, there's a cargo plugin `cargo unused` that can sometimes help find these.